File: single-language-analysis/dataset-files/python/test_smell_163.py
# Original URL: https://github.com/Sudistark/splunk-python/blob/e2c3e0a69ef29c4c8a6c4dcd59dd3c47ff620bd1/clilib/test_clilib.py#L35-L40

@pytest_mark_skip_conditional(reason="SPL-175665: Probably a regression or functional test now")
def testSourcetypes(args, fromCLI):
  paramsReq = (ARG_FILE,)
  paramsOpt = ()
  comm.validateArgs(paramsReq, paramsOpt, args)
  os.system("classify \"%s\"" % args[ARG_FILE])

File: single-language-analysis/dataset-files/java/TestSmell163.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell163 {

    @Test
    public void testSourcetypes() {
        String[] paramsReq = {ARG_FILE};
        String[] paramsOpt = {};
        comm.validateArgs(paramsReq, paramsOpt, args);
        try {
            Runtime.getRuntime().exec("classify \"" + args[ARG_FILE] + "\"");
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}